composer.json

```diff
@@ -49,6 +49,8 @@
             "php": "5.5.9"
         }
     },
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```

Comment:
Imo that's okay for the moment as Symfony 3.1 itself isn't stable yet. But yeah having a release for the APCu polyfill will be better.